### ISSUE TITLE
Trims switch assignment

### ISSUE THREAD

**Issue by yellowbird911** [FrSky-OS/discussion#72]
In the trim step menu it would be nice if a trim switch was assignable to any channel you wished.  The Jeti does this and allows you to assign a trim switch to say the nose wheel.  On the jets I fly this is probably the most common trim I adjust.

**Comment by AloftHobbies** [comment]
Good point. I use trims for a number of different uses. Often times in OpenTX I will use the throttle trim for different things based on a switch position. I don't think we will be able to have that functionality, but I do like using the trims for different functions. Some folks like cross trims.
